app/views/govuk_component/search.raw.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  appearance_class ||= "on-light-background"
+  layout_class ||= ""
```

Comment:
We only have one size variant at the moment (large) so I've implemented that as a flag. If we have more than one in the future we'd perhaps need another mechanism but there currently are no other sizes I'm aware of on gov.uk.